src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,23 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
```

Comment:
make it simple, just use `Props` name everywhere
```suggestion
interface Props {
```